### PR TITLE
Release notes 2.4.1

### PR DESCRIPTION
**Pull request by AnushaY1916**

### CODE REVIEW

File: release-notes/v2.4.1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Known Issues
+
+The following table lists the known issues for the HPE CSI Driver for Kubernetes v2.4.1. Please note that `Known Issues` from previous releases are still applicable with suggested workarounds if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|
+|--|---------|-----|-----------|
+|CON-3051|csi.k8s|Store CHAP credentials in K8s Secret rather than a ConfigMap||
+|CON-2867|csi.k8s|Unable to import offline vvols in Kubernetes cluster||
+|CON-2549|csi.k8s|Support volume expansion for RWX PVCs||
```
Comment (datamattsson) [inline release-notes/v2.4.1.md:33]: I don't think we should mention potential work in progress items here.

File: release-notes/v2.4.1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.4.1.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-3005|csi.k8s|RWX support for raw block volumes in Primera||
+|CON-3027|csi.k8s|ARM64 images support for the Nimble Container Storage Provider||
+|CON-3052|csi.k8s|Enumerate capabilities and volume types in OpenShift Security Context Constraints|Updated the SCOD documenatation|
+|CON-3020|csi.k8s|3PAR Container Storage Provider is unable to figure out the least used CPG when the user isn't supplying any CPG in the StorageClass or in the inline volume|Fixed the compute CPG algorithm for Arcus|
+|CON-3012|csi.k8s|Support Alletra Storage MP with 3PAR CSP||
+|CON-3009|csi.k8s|ARM64 images support for the CSI driver||
```
Comment (datamattsson) [inline release-notes/v2.4.1.md:20]: This is a dupe from line item #2?

File: release-notes/v2.4.1.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.4.1.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-3005|csi.k8s|RWX support for raw block volumes in Primera||
+|CON-3027|csi.k8s|ARM64 images support for the Nimble Container Storage Provider||
+|CON-3052|csi.k8s|Enumerate capabilities and volume types in OpenShift Security Context Constraints|Updated the SCOD documenatation|
+|CON-3020|csi.k8s|3PAR Container Storage Provider is unable to figure out the least used CPG when the user isn't supplying any CPG in the StorageClass or in the inline volume|Fixed the compute CPG algorithm for Arcus|
```
Comment (datamattsson) [inline release-notes/v2.4.1.md:18]: Arcus is an internal code name.

File: release-notes/v2.4.1.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+|CON-3005|csi.k8s|RWX support for raw block volumes in Primera||
+|CON-3027|csi.k8s|ARM64 images support for the Nimble Container Storage Provider||
+|CON-3052|csi.k8s|Enumerate capabilities and volume types in OpenShift Security Context Constraints| Updated the SCOD documenatation|
+|CON-3020|csi.k8s|3PAR Conatiner Storage Provider unable to figure out the least used CPG for Alletra MP||
+|CON-3012|csi.k8s|Support Alletra Storage MP with 3PAR CSP||
+|CON-3009|csi.k8s|ARM64 images support for the CSI driver||
+|CON-1743|csi.k8s|Support foreign StorageClass for multi-writers|Made enhancement to handle `nfsForeignStorageClass` parameter in storage class|
+|CON-3042|csi.k8s|Support and conform to SLE Micro||
+|CON-3044|csi.k8s|Removed glide(a package managing utility in GoLang) in common-host-libs and common-host-utils|Use go.mod to manage dependencies|
+
```
Comment (dileepds) [inline release-notes/v2.4.1.md:23]: Please include the issue. CON-3072  also, the fix for Arcus R3 connectivity issue

**Pull request closed without merging**